MyLibrary/Sources/SponsorFeature/Sponsors.swift

```diff
@@ -43,7 +43,7 @@ public struct SponsorsList {
         state.sponsors = try! dataClient.fetchSponsors()
         return .none
 
-      case let .view(.sponsorTapped(sponsor)):
+      case .view(.sponsorTapped(let sponsor)):
```

Comment:
What rule of swift-fomat is applied about precedence of `let` keyword?

Reply:
this rule is [UseLetInEveryBoundCaseVariable](https://github.com/apple/swift-format/blob/main/Documentation/RuleDocumentation.md#useletineveryboundcasevariable)
swift-format doesn't format this rule.